src/ccl_core.c

```diff
@@ -226,6 +226,12 @@ ccl_physical_constants ccl_constants = {
    */
   0.71611,
 
+  /**
+   * TNCDM: repeat the value to preserve API because we renamed TNCDM to T_ncdm.
+   * TODO: Remove for CCLv3.
+   */
+  0.71611,
+
```

Comment:
again, why are you renaming anything in the constants, if you're gonna remove them in v3 anyway?

Reply:
To preserve API if someone tries to do `ccl.physical_constants.TNCDM` vs the new one `ccl.physical_constants.T_ncdm`.